### ISSUE TITLE
Outlook issue with unicode characters in emails sent from FlowCrypt

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-browser#3504]
ref: https://mail.google.com/mail/u/human@flowcrypt.com/#inbox/FMfcgxwLsdHTzqXNjJktQpqJwmGVvDWM

I am already late on this support thread, so I need to do this very soon. I'm going to reproduce this on Windows with Outlook tomorrow and figure out where the source of the problem is. Either it's FlowCrypt's mistake, in which case I'll make a patch or try to, or it's Outlook's, and we report it to them and inform the user.

Probably it's us, but either way I'll work on this and post updates soon.

**Comment by seisvelas** [comment]
This problem goes like this:

1. Send an encrypted email with unicode characters to a recipient
2. The recipient opens the email in Outlook (everything is fine still)
3. The recipient replies. 
4. In the quoted reply in Outlook, unicode characters break!

So maybe the problem isn't us, but I should know for sure tomorrow.

**Comment by tomholub** [comment]
Sounds like the problem may be Outlook indeed. FlowCrypt used to have the same problem, we had to explicitly code for it. However, maybe we can learn something from the way Outlook itself does it, and update it in a way that doesn't break it (assuming Outlook also uses unicode)

**Comment by seisvelas** [comment]
Sorry I took so long - Outlook is not free either on Windows or Mac. Well, Outlook webmail is, but to use the desktop program you need a paid Microsoft 365 account. 

Should I just get the subscription? It might be nice to have the subscription for when Outlook compatibility situations like this pop up.

**Comment by tomholub** [comment]
I should set up one such account for general use. I'll let you know once I have it.

**Comment by tomholub** [comment]
I've made and shared an account for testing

**Comment by tomholub** [comment]
I think the plugin may be confused? Or maybe it's referring to the initial issue

**Comment by seisvelas** [comment]
I think you changing the name of the issue made it reevaluate the original issue body (since that counts as an edit, and it scans for edits). In any case, it's detecting the support email which is fixed in the newer version of the Action (the way it's used in the Android repo), I've made a PR to use the newer Action: https://github.com/FlowCrypt/flowcrypt-browser/pull/3664

**Comment by tomholub** [comment]
The Office subscription I purchased was for business and cannot be used with a personal account (flowcrypt.compatibility@outlook.com).

I don't have time to set up a company account on outlook. They don't seem to offer a personal subscription for Outlook app itself. I tried to purchase Outlook 2019 app for personal or business use separately, but they only support paysafecard for payment which seems to want me to go to a physical place to pay.

It seems I'm unable to buy Outlook, so I'm closing this as wontfix.
